### DISCUSSION TITLE
What other Terminal to use in Windows

### DISCUSSION THREAD

**Discussion by VictoryWekwa** [EddieHubCommunity/support#182] [Q&A]
I tried using babun but I keep getting error when I try to run install.bat, the error says `Make sure you typed the name correctly and try again.`

Who have suggestions on what other terminal to use in windows.

**Comment by eddiejaoude** [comment]
I dont use windows, but hopefully someone else in the community can help

**Comment by mikeysan** [comment]
What's wrong with using the built-in command prompt or PowerShell terminal?

**Reply by VictoryWekwa** [reply]
I don't know if I can use those to connect to a local server.
I will have to check it out.
Thank you

**Comment by rperry99** [comment]
Hey @VictoryWekwa I use Git Bash as my terminal on windows. So far, I have never had any issues with errors or anything. You can view more information on this [here](https://gitforwindows.org/).

**Reply by VictoryWekwa** [reply]
Okay. Thank you, will check it out.

**Comment by AllanRegush** [comment]
Powershell, git bash, [cmder](https://cmder.net/)

**Comment by AllanRegush** [comment]
Looks like babun is no longer being supported https://github.com/babun/babun/issues/868

**Comment by mikeysan** [comment]
I'm going to have to try git bash, but to my knowledge, git bash is an emulation of the bash shell and may not have access to run a .bat script, which is Windows native. I could be wrong. I would advice using the PowerShell terminal or Windows Command Line (cmd).

If you are unable to perform the installing using cmd, then let us know and we'll assist further.

**Comment by AllanRegush** [comment]
Are you looking for a cygwin installer? https://github.com/vegardit/cygwin-portable-installer

**Reply by VictoryWekwa** [reply]
Yeah.
Babun comes with cygwin.
Will have to try out Powershell first.
Thank you

**Reply by mikeysan** [reply]
Hey @VictoryWekwa How did you get on with testing PowerShell or any of the other suggestions?

**Comment by amendezm** [comment]
@VictoryWekwa I use Windows Terminal

![image](https://user-images.githubusercontent.com/26444448/179433924-298dc191-fbf9-43b9-9ef4-a1ca754eb4f5.png)

![image](https://user-images.githubusercontent.com/26444448/179433926-4c547a69-b421-4eb9-b3cd-d89f28b331b3.png)

**Reply by mikeysan** [reply]
We all use this now, I should hope :)

**Comment by prathikshetty2002** [comment]
You can install the WSL Linux terminal ubuntu one is recommended. It is great and lets you do any work seamlessly. 

You can get it on the Microsoft store: https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV

Here you can read their docs : https://ubuntu.com/wsl

You can tweak it according to your choice & it's flexible too.

![Screenshot 2022-08-30 233957](https://user-images.githubusercontent.com/63772910/187511780-7dac03c0-0810-4f34-9986-72de862437ad.png)

Must try once WSL2 is a great tool for developers🙌💡

**Reply by mikeysan** [reply]
WSL is a great tool but it has nothing to do with the question that was asked. It is always helpful to read the original question when responding to an old thread.

**Comment by suitupstan** [comment]
Windows Terminal is a cool option, pair it with WSL and you get a beast of a terminal for your work machine

**Comment by yaswanthteja** [comment]
> I tried using babun but I keep getting error when I try to run install.bat, the error says `Make sure you typed the name correctly and try again.`
> 
> Who have suggestions on what other terminal to use in windows.

I like to use windows Terminal, git bash & we can also use WSL
